frontend/src/features/whats-new/FeatureUpdateList.ts

```diff
@@ -1,8 +1,9 @@
 import dedent from 'dedent'
 import { JsonObject, RequireExactlyOne } from 'type-fest'
 
-import Animation1 from './assets/1-dnd.json'
+import Animation1 from './assets/1-payments.json'
 import Animation2 from './assets/2-search-and-filter.json'
+import Animation3 from './assets/3-dnd.json'
```

Comment:
Same comment on not naming assets with positions in their name, but also same comment that there's no need to change that now in the PR.